edge/pkg/devicetwin/dtcontext/dtcontext.go

```diff
@@ -1,19 +1,19 @@
 package dtcontext
 
 import (
-	"github.com/kubeedge/kubeedge/beehive/pkg/common/config"
-	"github.com/kubeedge/kubeedge/beehive/pkg/common/log"
-	"github.com/kubeedge/kubeedge/beehive/pkg/core"
-	"github.com/kubeedge/kubeedge/beehive/pkg/core/context"
-	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/common/config"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/common/log"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/core/context"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/core/model"
```

Comment:
please move the github.com/* section together with the following lines that importing github.com/*

Reply:
fixed